src/components/ViewItem/ViewItem.js

```diff
@@ -27,15 +27,15 @@ const sortMap = {
   time: loan => moment(loan.dueDate).format('hh:mm a'),
 };
 
-const visibleColumns = ['no', 'barcode', 'title', 'loanPolicy', 'dueDate', 'time', ' '];
+const visibleColumns = ['no', 'barcode', 'title', 'loanPolicy', 'dueDate', 'time', 'details'];
 
 const columnWidths = {
```

Comment:
![805](https://user-images.githubusercontent.com/24813219/210778125-426bc5ab-4779-4dfa-bfb2-f9051f64bb1d.PNG)
May be this is my local problem, but how i understood time should be placed in 1 line and we don't need scroll in the bottom.
Could you please double check render?